modules/t/translation.t

```diff
@@ -343,4 +344,23 @@ ok(!scalar(@alt_tls));
 # Test generic_count(), inherited method from BaseAdaptor
 is($ta->generic_count(), @{$ta->list_dbIDs()}, "Number of features from generic_count is equal to the number of dbIDs from list_dbIDs");
 
+#56-57
+#
+# Handling stop codon readthrough edit
+#
+
+my $tra_scrt = $db->get_TranscriptAdaptor();
```

Comment:
You'll make more friends if you name variables less cryptically. This is a transcript adaptor. $adaptor, $transcript_adaptor, and $tra would all be less cryptic.